src/utils/source.js

```diff
@@ -156,7 +156,7 @@ export function getFilename(source: Source) {
 
   const { filename } = getURL(source);
 
-  return filename;
+  return getRawSourceURL(filename);
```

Comment:
should we call it before getURL i.e. `getURL(getRawSourceURL(source))`?

we could also have a param:

`getURL(..., {raw: true})`

Reply:
are there cases where we're using getRawSourceURL and wouldnt need it anymore after this change?
are there cases whee we're calling getFilename and dont expect it to be raw?

Reply:
Per your comments:

1.  We can't pass `getRawSourceURL` a source because it's used in cases where we don't have a `source` object
2.  We're currently using `getRawSourceURL` in places where it's reasonable to keep the function independent, i.e. copying source URL 
3.  It doesn't appear so